examples/guided.py

```diff
@@ -263,42 +349,55 @@ def perform_installation_steps():
 		with archinstall.Filesystem(archinstall.arguments['harddrive'], mode) as fs:
 			# Wipe the entire drive if the disk flag `keep_partitions`is False.
 			if archinstall.arguments['harddrive'].keep_partitions is False:
-				fs.use_entire_disk(root_filesystem_type=archinstall.arguments.get('filesystem', 'btrfs'))
-			
+				fs.use_entire_disk(
+					root_filesystem_type=archinstall.arguments.get('filesystem', 'btrfs')
```

Comment:
Disclaimer (going to be a lot of these), this following is my personal opinion:

This decreases readability.